src/runtime/plugin.dev.ts

```diff
@@ -3,21 +3,51 @@ import type { NitroAppPlugin } from "nitropack";
 import { useRuntimeConfig } from "#imports";
 
 export default <NitroAppPlugin>function (nitroApp) {
-  let _proxy: ReturnType<typeof getBindingsProxy>;
+  let _proxy: ReturnType<typeof getPlatformProxy>;
 
   nitroApp.hooks.hook("request", async (event) => {
     // Lazy initialize proxy when first request comes in
     if (!_proxy) {
-      _proxy = getBindingsProxy().catch((error) => {
+      _proxy = getPlatformProxy().catch((error) => {
         console.error("Failed to initialize wrangler bindings proxy", error);
-        return { bindings: {}, dispose: () => Promise.resolve() };
+        return {
+          env: {},
+          cf: {},
+          ctx: {
+            waitUntil() {},
+            passThroughOnException() {},
+          },
+          caches: {
+            open(): Promise<Cache> {
+              const result = Promise.resolve(new Cache());
+              return result;
+            },
+            get default(): Cache {
+              return new Cache();
+            },
+          },
+          dispose: () => Promise.resolve(),
+        } as unknown as ReturnType<typeof getPlatformProxy>;
       });
     }
 
+    const proxy = await _proxy;
+
+    // Inject the cf object onto the context
+    (event.context as any).cf = proxy.cf;
+
+    // Inject the waitUntil onto the context
+    (event.context as any).waitUntil = proxy.ctx.waitUntil;
+
+    const request = new Request(`http://localhost${event.path}`);
```

Comment:
@pi0 sorry if it's a dumb question, but here, could I get the address + port of the current nitro server?